rdc_libs/CMakeLists.txt

```diff
@@ -61,7 +61,9 @@ set(RDC_LIB "rdc")
 set(RDC_ROCR_LIB "rdc_rocr")
 set(RDC_ROCP_LIB "rdc_rocp")
 set(RDC_RVS_LIB "rdc_rvs")
-set(RDCCLIENT_LIB "rdc_client")
+if(BUILD_STANDALONE)
```

Comment:
By default, we should build everything. Only if the BUILD_STANDALONE=OFF, it will exclude rdc_client.

Reply:
BUILD_STANDALONE is on by default, so this isn't an issue